### PR TITLE
Hypospray buffs & polish from Nova Sector

### PR DESCRIPTION
**Pull request by Superhats**
## About The Pull Request
Mostly direct port of https://github.com/NovaSector/NovaSector/pull/1283 from nova sector, with a couple changes to make them no longer fit in pockets/medical belt.

## Why It's Good For The Game
Gives some love to hyposprays, to make them slightly more competitive over just carrying a bunch of medical gels. Also generally just nice looking sprite and GAGS is really cool.

## Proof Of Testing
image

## Changelog

:cl:VrazzleDazzle
balance: Hyposprays received an array of balance changes.
image: New hypo sprites courtesy of Nova Sector.
/:cl:

### CODE REVIEW

**Comment by Superhats** [comment]
Holy fucking bingle it compiled

**Comment by YakumoChen** [comment]
> small hypospray kits only have seven slots of storage but can fit in a pocket

That's some pretty massive storage creep

**Comment by YakumoChen** [comment]
> while the deluxe hypospray kits have three rows of storage at the cost of no longer being pocketable. Both still fit in medical belts.

Holy storage creep batman

**Comment by Superhats** [comment]
> > small hypospray kits only have seven slots of storage but can fit in a pocket
> 
> That's some pretty massive storage creep

I mean, the mini medkit is in the game, which has five slots, fits everything and not just hypovials, and can be bought from cargo. Hypokits fitting in belts I was under the impression that it was already a thing, I'm gonna make that not a thing.

**Comment by Superhats** [comment]
Keeping in mind the kits hold **exclusively** hypovials, and there's really no realistic scenario where you need the amount of chems from more than one kit.

**Comment by YakumoChen** [comment]
> I mean, the mini medkit is in the game, which has five slots,

That's also skyrat storage creep and needs to go at some point

>Two pockets full of hypokits is 700u chem, which is more than any doctor will ever need. 

That's exactly the problem yes, doctors don't need to be able to hold all the chems for their shift it one pocket (or two)

**Comment by Superhats** [comment]
There. The pocket stuff's been executed

File: modular_skyrat/modules/hyposprays/code/hypospray_kits.dm [hunk 1]
```diff
@@ -35,7 +33,7 @@
 	. = ..()
 	if(!length(case_designs))
 		populate_case_designs()
-	atom_storage.max_slots = 7
+	atom_storage.max_slots = 12
```
Comment (YakumoChen) [inline modular_skyrat/modules/hyposprays/code/hypospray_kits.dm:36]: You removed the pocket storage creep then almost doubled it's capacity?

Comment (Superhats) [reply modular_skyrat/modules/hyposprays/code/hypospray_kits.dm:36]: This is on Bubber. The storage was reduced to 7 on nova to compensate for the pocket thing.

**Comment by projectkepler-ru** [comment]
obligatory (do this upstream)

File: modular_skyrat/modules/hyposprays/code/hypospray_kits.dm (hunk 1 above)
Comment (YakumoChen) [reply modular_skyrat/modules/hyposprays/code/hypospray_kits.dm]: Why does it need 12 slots instead of the usual 7 every other box of anything has?

Comment (Superhats) [reply modular_skyrat/modules/hyposprays/code/hypospray_kits.dm]: Twelve slots is what we currently have. I didn't increase it, I reverted Nova's nerf.

**Comment by Superhats** [comment]
I really don't feel like dealing with Skyrat's upstream so I'm gonna be closing this (and my other PR https://github.com/Bubberstation/Bubberstation/pull/1333)

**Pull request closed without merging**